core/src/createTextMaskInputElement.js

```diff
@@ -25,160 +75,135 @@ export default function createTextMaskInputElement({
     providedMask = providedMask.mask
   }
```

Comment:
Yeah, I was mostly attempting to have the only state be the previous conformed value.
I can extract that stuff to a separate function to pre-process, just only wanted the RN mask to have one thing to call but I can have it call both things